src/multicellular_stage/systems/IntercellularMatrixSystem.cs

```diff
@@ -66,19 +66,31 @@ private static void AddIntercellularConnection(in Entity entity, ref Intercellul
 
         Vector3 targetRelativePos;
 
+        Quaternion ourRotation;
+        Quaternion targetRotation = Quaternion.Identity;
+
         if (parentEntity == colony.Leader)
         {
-            targetRelativePos = -entity.Get<AttachedToEntity>().RelativePosition;
+            var ourAttachedPosition = entity.Get<AttachedToEntity>();
```

Comment:
```suggestion
            ref var ourAttachedPosition = ref entity.Get<AttachedToEntity>();
```
Components need to be fetched like this to avoid extra memory copies.